src/lib/storage/storage_manager.cpp

```diff
@@ -10,38 +10,47 @@
 namespace opossum {
 
 StorageManager& StorageManager::get() {
-  return *(new StorageManager());
-  // A really hacky fix to get the tests to run - replace this with your implementation
+  static StorageManager _instance;
```

Comment:
We only use a leading underscore for private/protected members/methods but not for local variables, see `CONTRIBUTING.md`.